src/zjs_dgram.c

```diff
@@ -217,22 +215,19 @@ static void udp_sent(struct net_context *context, int status, void *token,
     DBG_PRINT("udp_sent: %p, st=%d udata=%p\n", context, status, user_data);
 
     if (user_data) {
-        jerry_value_t args[1] = {ZJS_UNDEFINED};
+        ZVAL rval = ZJS_UNDEFINED;
         if (status != 0) {
             char errbuf[8];
             snprintf(errbuf, sizeof(errbuf), "%d", status);
-            args[0] = zjs_standard_error(NetworkError, errbuf);
+            rval = zjs_standard_error(NetworkError, errbuf);
             // We need error object, not error value (JrS doesn't allow to
             // pass the latter as a func argument).
-            jerry_value_clear_error_flag(&args[0]);
+            jerry_value_clear_error_flag(&rval);
         }
 
         zjs_callback_id id = zjs_add_callback_once((jerry_value_t)user_data,
-                                                   ZJS_UNDEFINED,
-                                                   NULL,
-                                                   NULL);
-        zjs_signal_callback(id, args, sizeof(args));
-        jerry_release_value(args[0]);
+                                                   ZJS_UNDEFINED, NULL, NULL);
+        zjs_signal_callback(id, &rval, 1);
```

Comment:
np, this happens, my thinking is "how can we avoid this issue, as it may happen again for other folks", hence ideas https://github.com/01org/zephyr.js/pull/842#issuecomment-289207453 (which might not work, at least we try to think about it).